src/vdaf/hits.rs

```diff
@@ -437,92 +437,87 @@ impl<I: Idpf<2, 2>> Aggregator for Hits<I> {
         })
     }
 
+    fn prepare_preprocess<M: IntoIterator<Item = Self::PrepareMessage>>(
+        &self,
+        inputs: M,
+    ) -> Result<Self::PrepareMessage, VdafError> {
```

Comment:
```suggestion
    fn prepare_preprocess<M: IntoIterator<Item = HitsPrepareMessage>>(
        &self,
        inputs: M,
    ) -> Result<HitsPrepareMessage, VdafError> {
```
...and then this is good to go